test/util/fixtures/ClientServiceFixtures.java

```diff
@@ -0,0 +1,54 @@
+package util.fixtures;
+
+import entities.Client;
+import enums.Roles;
+import enums.StatusEnum;
+import java.time.Instant;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public final class ClientServiceFixtures {
+  public static final Long FAKEID = Long.valueOf(10L);
+
+  public static List<Client> buildClients(Integer size) {
+    List<Client> clients =
+        IntStream.range(0, size)
+            .mapToObj(i -> ClientServiceFixtures.buildClient())
+            .collect(Collectors.toList());
+    return clients;
+  }
+
+  public static Client buildClient() {
+    return buildClient(null);
+  }
+
+  public static Client buildClient(Client clientExample) {
+    var client = new Client();
+    client.setName(Optional.ofNullable(clientExample).map(Client::getName).orElse("testing-name"));
```

Comment:
You are missing setting the client id as well

Reply:
solved